lib/common/api/deprecate.ts

```diff
@@ -34,9 +34,27 @@ const deprecate: ElectronInternal.DeprecationUtil = {
     }
   },
 
+  // remove a function with no replacement
+  removeFunction: (fn, removedName) => {
+    // if the function has already been removed, warn about it
+    if (!fn) {
+      deprecate.log(`Unable to remove function '${removedName}' from an object that lacks it.`)
```

Comment:
If the function is gone, its deprecation handling wrapper will also have been removed; we can throw an error here and bail early if the function is somehow invalid but there's no need to return a function i don't think.

Reply:
Yes, should only affect us (and as soon as everything is typescript, it won't compile anyway). I'll throw an error...